python/foxglove-sdk/python/foxglove/layouts/panels.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from typing import Literal
+
+from . import Panel
+
+
+class MarkdownPanel(Panel):
+    def __init__(self, *, markdown: str | None = None, font_size: int | None = None):
+        super().__init__("Markdown", markdown=markdown, fontSize=font_size)
+
+
+class RawMessagesPanel(Panel):
```

Comment:
how would one support [schema converters](https://docs.foxglove.dev/docs/visualization/panels/raw-messages#message-conversion) in this scheme?

Reply:
The selected converter is kept in memory rather than in the layout.